Source/Scene/ClassificationModel.js

```diff
@@ -0,0 +1,1209 @@
+         * }).otherwise(function(error){
+         *   window.alert(error);
+         * });
+         *
+         * @see ClassificationModel#ready
+         */
+        readyPromise : {
+            get : function() {
+                return this._readyPromise.promise;
+            }
+        },
+
+        /**
+         * Returns true if the model was transformed this frame
+         *
+         * @memberof ClassificationModel.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @private
+         */
+        dirty : {
+            get : function() {
+                return this._dirty;
+            }
+        },
+
+        extensionsUsed : {
+            get : function() {
+                if (!defined(this._extensionsUsed)) {
+                    this._extensionsUsed = ModelUtility.getUsedExtensions(this.gltf);
+                }
+                return this._extensionsUsed;
+            }
+        },
+
+        extensionsRequired : {
+            get : function() {
+                if (!defined(this._extensionsRequired)) {
+                    this._extensionsRequired = ModelUtility.getRequiredExtensions(this.gltf);
+                }
+                return this._extensionsRequired;
+            }
+        },
+
+        /**
+         * Gets the model's up-axis.
+         * By default models are y-up according to the glTF spec, however geo-referenced models will typically be z-up.
+         *
+         * @memberof ClassificationModel.prototype
+         *
+         * @type {Number}
+         * @default Axis.Y
+         * @readonly
+         *
+         * @private
+         */
+        upAxis : {
+            get : function() {
+                return this._upAxis;
+            }
+        },
+
+        /**
+         * Gets the model's triangle count.
+         *
+         * @private
+         */
+        trianglesLength : {
+            get : function() {
+                return this._trianglesLength;
+            }
+        },
+
+        /**
+         * Gets the model's geometry memory in bytes. This includes all vertex and index buffers.
+         *
+         * @private
+         */
+        geometryByteLength : {
+            get : function() {
+                return this._geometryByteLength;
+            }
+        },
+
+        /**
+         * Gets the model's texture memory in bytes.
+         *
+         * @private
+         */
+        texturesByteLength : {
+            get : function() {
+                return 0;
+            }
+        },
+
+        /**
+         * Gets the model's classification type.
+         * @memberof ClassificationModel.prototype
+         * @type {ClassificationType}
+         */
+        classificationType : {
+            get : function() {
+                return this._classificationType;
+            }
+        }
+    });
+
+    var aMinScratch = new Cartesian3();
+    var aMaxScratch = new Cartesian3();
+
+    function computeBoundingSphere(model) {
```

Comment:
Can this be a common function?

Reply:
This function is different enough from model that I didn't create a common function. The only part I could pull out is finding the min and max.